src/main/java/org/launchcode/techjobs/persistent/controllers/EmployerController.java

```diff
@@ -25,6 +39,7 @@ public String processAddEmployerForm(@ModelAttribute @Valid Employer newEmployer
                                     Errors errors, Model model) {
 
         if (errors.hasErrors()) {
+            model.addAttribute("employer", "Add Employer");
             return "employers/add";
         }
 
```

Comment:
To make the `testNewEmployerIsSaved` pass, you'll need to save `newEmployer` in the `employerRepository`.